### ISSUE TITLE
Node.js 16 actions are deprecated. Please update the following actions to use Node.js 20: azure/docker-login@v1.0.1.

### ISSUE THREAD

**Issue by loisra01** [Azure/docker-login#67]
Node.js 16 actions are deprecated. Please update the following actions to use Node.js 20: azure/docker-login@v1.0.1. For more information see: https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/.

**Comment by YvesEarnix** [comment]
Look on the [Docker login action](https://github.com/marketplace/actions/docker-login) created by docker

```
name: ci

on:
  push:
    branches: main

jobs:
  login:
    runs-on: ubuntu-latest
    steps:
      -
        name: Login to ACR
        uses: docker/login-action@v3
        with:
          registry: <registry-name>.azurecr.io
          username: ${{ secrets.AZURE_CLIENT_ID }}
          password: ${{ secrets.AZURE_CLIENT_SECRET }}

**Comment by lalten** [comment]
https://github.com/Azure/login/releases/tag/v2.0.0 was released recently with a Nodejs 20 upgrade. Can we get this action upgraded as well please @BALAGA-GAYATRI?

**Comment by mxschmitt** [comment]
Might be relevant for others, I usually do the following nowadays:

```yml
name: Example
on:
  push:
    branches:
      - main
permissions:
  id-token: write # This is required for azure/login
  contents: read  # This is required for actions/checkout
jobs:
  index_firefox:
    name: Example
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v4
    - uses: azure/login@v2
      with:
        client-id: ${{ secrets.AZURE_CLIENT_ID }}
        tenant-id: ${{ secrets.AZURE_TENANT_ID }}
        subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
    - name: Login to ACR via OIDC
      run: az acr login --name example
    - name: puch container
      run: docker push example.azurecr.io/example:latest
    # Do e.g. a restart of the webapp
    - name: restart ffsearch
      run: az webapp restart ...
```

This combines [OpenID secret-free login](https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-azure) with e.g. ACR.

**Comment by lgmorand** [comment]
PR merged

**Comment by joshmedeski** [comment]
I ran into an issue, see https://github.com/Azure/docker-login/issues/68

**Comment by lgmorand** [comment]
Normally fixed by #69
